apbs/chemistry/atom.py

```diff
@@ -45,11 +104,14 @@ def __repr__(self):
             self.charge,
         )
 
-    def euclidian_dist2(
-        self, other: Union["Atom", Coordinate, np.ndarray]
-    ) -> float:
+    def euclidian_dist2(self, other: Union["Atom", Coordinate, np.ndarray]) -> float:
```

Comment:
How does Sphinx deal with mixed docstrings vs. function signature types of annotation?  In particular, does `-> float` replace `:returns:` and `:rtype:`?

Reply:
I added the returns and rtype. Since it only returns float, I think the mixed signature works.